### PR TITLE
Campsite setup tests

### PR DESCRIPTION
**Pull request by t-keazirian**
Went on a bit of a refactor rampage within all my tests:

- Learned more about grouping tests today on my other PR so I wanted to put that into practice in the rest of the tests
- Learned about setUp() within tests so added that to all tests that it applies to
- changed `assertEqual(True/False, value)` to `assertTrue/False(value)` for readability
- Changed a few names of tests for clarity/consistency

### CODE REVIEW

File: test/test_board.py [hunk 1]
```diff
@@ -4,163 +4,85 @@
...
         self.assertEqual(1, total_marks_on_board)
 
-    def test_board_is_marked_with_user_selection_when_no_marks_and_X_is_going_first(
-        self,
-    ):
-        board = Board()
-        current_board = ["1", "2", "3", "4", "5", "6", "7", "8", "9"]
-        new_board_with_marks = ["X", "2", "3", "4", "5", "6", "7", "8", "9"]
-        user_input = 1
+    def test_mark_board_marks_board_with_user_selection(self):
```
Comment (Aoxian) [inline test/test_board.py:50]: While this is good, I still suggest keeping the "arrange -> act -> assert" pattern in your tests. This helps to keep it human readable.  

```python
    def test_mark_board_marks_board_with_user_selection(self):
        test_mark_one = "X"
        test_mark_two = "O"
        test_mark_three = "🤡"
        expected_board = [test_mark_one, test_mark_two, test_mark_one, test_mark_three, "5", "6", "7", "8", "9"]

        self.board.mark_board(1, self.starter_board, test_mark_one)
        self.board.mark_board(2, self.starter_board, test_mark_two)
        self.board.mark_board(3, self.starter_board, test_mark_one)
        self.board.mark_board(4, self.starter_board, test_mark_three)

        self.assertEqual(self.board, expected_board)
```

File: test/test_game.py [hunk 2]
```diff
@@ -10,102 +10,84 @@
...
-        message = message.choose_symbol_player_one()
+        test_message = self.message.choose_symbol_player_one()
 
-        result = game.set_player_symbol(message)
+        result = self.game.set_player_symbol(test_message)
 
         self.assertEqual(expected_symbol, result)
 
     @patch("builtins.input", side_effect=["2"])
     def test_set_player_two_symbol_returns_symbol_with_input_2(self, mock_input):
```
Comment (Aoxian) [inline test/test_game.py:70]: this test has the extra spacing between "arrange -> act -> assert"; while this is my preference, the rest of the tests in this project do not use it. Pick a way, and stick with it throughout.

File: test/test_rules.py [hunk 3]
```diff
@@ -13,53 +13,53 @@ def test_horizontal_rows_return_true_when_three_in_a_row(self):
         test_middle_row = ["1", "2", "3", "X", "X", "X", "7", "8", "9"]
         test_bottom_row = ["1", "2", "3", "4", "5", "6", "X", "X", "X"]
 
-        self.assertEqual(True, self.rules.is_winner_horizontal(test_top_row))
-        self.assertEqual(True, self.rules.is_winner_horizontal(test_middle_row))
-        self.assertEqual(True, self.rules.is_winner_horizontal(test_bottom_row))
+        self.assertTrue(self.rules.is_winner_horizontal(test_top_row))
```
Comment (Aoxian) [inline test/test_rules.py:16]: nice

**Review by Aoxian — APPROVED** [review]
Approved with comments.

🧹

File: test/test_game.py (hunk 2 above)
Comment (t-keazirian) [reply test/test_game.py:70]: I like it that way better too - I'll change them all to be consistent!

**Pull request merged**